UnityProject/Assets/Scripts/Objects/Chemistry/ChemMaster.cs

```diff
@@ -0,0 +1,329 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Chemistry.Components;
+using Mirror;
+using UnityEngine;
+using Systems.Electricity;
+using Items;
+
+namespace Chemistry
+{
+	/// <summary>
+	/// Main component for ChemMaster, or Chemical Master™.
+	/// </summary>
+	public class ChemMaster : NetworkBehaviour, ICheckedInteractable<HandApply>, IAPCPowerable
```

Comment:
Hmm couldn't find anything that would require inheriting from `NetworkBehaviour`. Did I miss something?
```suggestion
	public class ChemMaster : MonoBehaviour, ICheckedInteractable<HandApply>, IAPCPowerable
```

Reply:
That is a fair point, between Interaction Management and NetUI already handling the messaging.